docs/test_runners.md

```diff
@@ -0,0 +1,24 @@
+# Elección de `test runner`
+> Objetivo 4
+
+## Criterios
+Facilidad de uso, facilidad de instalación, velocidad.
+
+## Opciones
+> Tras leer en varios foros y teniendo en cuenta las características del modelo, las siguientes son las opciones que tendremos en cosideración pese a que haya muchas alternativas más.
+
+### Mocha
+* Fácil de usar, hay muchas dudas resueltas en foros y extensa documentación.
+* Fácil de instalar, pero no incluye una librería de aserciones, por lo que habría que instalarla aparte.
+* Es muy rápido, además de asíncrono.
+
+### Jest
+* Fácil de usar, hay mucha documentación online y su manual es muy completo.
+* Fácil de instalar, además, incluye una librería de aserciones.
+* Es rápido y asíncrono como en el caso anterior.
+
+## Elección
+
+Tras haber probado las dos opciones, nos llama la atención el funcionamiento de `Jest`, por ser más rápido, por la calidad de su manual, facilidad de uso y por incluir ya la libreria de aserciones.
```

Comment:
Hay al menos un par de opciones más con cierta popularidad y posiblemente media docena sin tanta popularidad. Debéis aprender a buscar exhaustivamente las herramientas que vayáis a usar, mediante los buscadores del repositorio de bibliotecas correspondientes, por ejemplo.

Reply:
:arrow_up: